docs/CostCenters.md

```diff
@@ -80,6 +84,29 @@ contxtSdk.facilities.costCenters
   .catch((err) => console.log(err));
 ```
 
+<a name="CostCenters+delete"></a>
+
+### costCenters.delete(costCenterId) ⇒ <code>Promise</code>
+
+Delete a cost center
+
+API Endpoint: '/costcenters/:costCenterId'
+Method: DELETE
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Fulfill**: <code>undefined</code>  
+**Reject**: <code>Error</code>
+
+| Param        | Type                | Description                                     |
+| ------------ | ------------------- | ----------------------------------------------- |
+| costCenterId | <code>string</code> | The id of the cost center (formatted as a UUID) |
```

Comment:
I've seen id capitalized and non-capitalized. Should we capitalize it here?

Reply:
Yes we should! Good catch.